CONTRIBUTING.md

```diff
@@ -129,6 +130,21 @@ If you'd like to contribute a post to layer5.io/blog, please open an Issue and s
 3. Follow the instructions included in the news template and name the new file after the title of the news article.
 4. Entries will be listed in chronological order automatically.
 
+## Contributing to Layer5's Sistent documentation
+
+If you'd like to contribute a post to layer5.io/projects/sistent/components. start by selecting the area/sistent label in the [#GitHub issue tracker](https://github.com/layer5io/layer5/labels/project%2Fsistent). We've streamlined the process by introducing a dynamic page creation workflow, simplifying the addition of new pages and ensuring a consistent structure for all contributions.
```

Comment:
First: don't write the link on the documentation(use link tag and use label here)
second: why its specific to components? anywhere near sistent project should be right? and this particular process could be for adding the components for sistent.

Reply:
I have updated contributing.md with label and general guidelines for sistent